ui/app/routes/allocations/allocation/task.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+const { Route, inject, Error: EmberError } = Ember;
+
+export default Route.extend({
+  store: inject.service(),
+
+  model({ name }) {
+    const allocation = this.modelFor('allocations.allocation');
+    if (allocation) {
```

Comment:
else?

Reply:
If allocation is falsey, the allocation route model hook will set the application error state.